### ISSUE TITLE
qSlicerWebWidget broken on macOS

### ISSUE THREAD

**Issue by sjh26** [Slicer/Slicer#5373]
The qSlicerWebWidget class throws OpenGL errors when created on macOS Slicer

## Steps to reproduce

1. Start Slicer
2. In the Python interactor run `webWidget = slicer.qSlicerWebWidget()`
3. In the error log, you will see this (or similar) error: `ERROR:gl_context:cgl.cc(118)] Error creating context`

## Expected behavior

The object should be created without errors.

## Environment
- Slicer version: Slicer-4.13.0-2021-01-05
- Operating system: Mac High Sierra (10.13.6) and Catalina (10.15.6)

**Comment by lassoan** [comment]
Is this new (appeared in today's release) or you experienced it with earlier Slicer versions, too?

**Comment by sjh26** [comment]
I will do some testing, I doubt its new.

**Comment by sjh26** [comment]
It is also present in the stable version

**Comment by pieper** [comment]
Yes, I get this error too on mac and it has existed for a while. As far as I can tell it is just a false alarm since the web widget works as expected (e.g. it can render slicer.org and a page with webgl2 content - I've never noticed anything broken).

**Comment by lassoan** [comment]
Rendering in the web widget is broken if light mode is used in the OS (https://github.com/Slicer/Slicer/issues/5118) but probably it is not related.
